src/data/markdown/docs/02 javascript api/11 k6-ws/20-Params.md

```diff
@@ -6,10 +6,12 @@ excerpt: 'Used for setting various WebSocket request-specific parameters such as
-| --------------------- | ------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
-| `Params.headers`      | object | Object with key-value pairs representing custom headers the user would like to add to the request.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
-| `Params.tags`         | object | Key-value pairs where the keys are names of tags and the values are tag values. Response time metrics generated as a result of the request will have these tags added to them, allowing the user to filter out those results specifically, when looking at results data.                                                                                                                                                                                                                                                                                                                          |
+| Name                  | Type   | Description |
+| --------------------- | ------ | ----------- |
+| `Params.compression`  | string | Empty by default, but when specified, WebSocket compression will be enabled with the specified algorithm. The only currently supported algorithm is `deflate`. |
+| `Params.jar`          | [http.CookieJar](https://k6.io/docs/javascript-api/k6-http/cookiejar/) | The cookie jar that will be used when making the initial HTTP request to establish the WebSocket connection. If empty, the [default VU cookie jar](https://k6.io/docs/javascript-api/k6-http/cookiejar-method) will be used. |
+| `Params.headers`      | object | Object with key-value pairs representing custom headers the user would like to add to the request. |
```

Comment:
Custom headers in key-value pairs the user would like to add to the request. Keys are header names, and values are header values.

Reply:
> Custom HTTP headers in key-value pairs the user would like to add to the initial HTTP request that establishes the WebSocket connection. Keys are header names, and values are header values.

Reply:
Better ;)